bciers/apps/registration/app/components/operations/registration/FacilityInformationForm.tsx

```diff
@@ -143,12 +138,10 @@ const FacilityInformationForm = ({
     <MultiStepBase
       allowBackNavigation
       baseUrl={`/register-an-operation/${operation}`}
-      baseUrlParams="title=Placeholder+Title"
```

Comment:
The wireframes don't show anything besides `Register an Operation` in the breadcrumbs, so we don't need a title placeholder atm